From: Colin Ian King <colin.i.king@gmail.com>
Subject: mm: Fix spelling mistake "commdandline" -> "commandline"
Date: Thu, 15 Feb 2024 09:05:44 +0000

There is a spelling mistake in a pr_warn_once message. Fix it.

Link: https://lkml.kernel.org/r/20240215090544.1649201-1-colin.i.king@gmail.com
Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Cc: Johannes Weiner <hannes@cmpxchg.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/memcontrol.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/memcontrol.c~mm-memcontrol-clarify-swapaccount=0-deprecation-warning-fix
+++ a/mm/memcontrol.c
@@ -7974,7 +7974,7 @@ static int __init setup_swap_account(cha
 	bool res;
 
 	if (!kstrtobool(s, &res) && !res)
-		pr_warn_once("The swapaccount=0 commdandline option is deprecated "
+		pr_warn_once("The swapaccount=0 commandline option is deprecated "
 			     "in favor of configuring swap control via cgroupfs. "
 			     "Please report your usecase to linux-mm@kvack.org if you "
 			     "depend on this functionality.\n");
_
